rosidl_cli/rosidl_cli/common.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def get_first_line_doc(any_type):
+    if not any_type.__doc__:
+        return ''
+    lines = any_type.__doc__.splitlines()
```

Comment:
Sounds good. See https://github.com/ros2/rosidl/pull/567/commits/3a75bcdbc3e2088ecd9929cdac7ad912b4de1dee.

Reply:
Hmm, that's not what I had in mind. The `if line.strip()`  at the end in my example would filter out all whitespace only strings, making the `if lines[0]`/`elif len(lines) > 1` checks unnecessary.

Now that rosidl has full control of the sub-commands, I can see why the feedback I gave before wouldn't matter. Since rosidl controls all the commands and extra whitespace in the docstrings wouldn't get passed the linters, the code that was here before was fine.

Reply:
Oh, yeah, I see now. I was trying to get rid of the duplicate `.strip()` and i missed the redundant checks :sweat_smile: 

I think 219869e captures your idea and satisfies my OCD. PTAL !

Reply:
Plus https://github.com/ros2/rosidl/pull/567/commits/b148c07b2922abbac67c23a6ea15a5c92221ec5a... Shouldn't code when tired.